tests/stress/grpc_stress/test_stress_grpc.py

```diff
@@ -0,0 +1,70 @@
+from pytest_testconfig import config as testconfig
+import multiprocessing as mp
+import random
+
+from tests.convenience import sleep_print_backwards
+from tests.queries import find_error_log_msgs
+from tests.test_bs import setup_network, add_curl, setup_bootstrap, start_poet, setup_clients, wait_genesis
+from tests.tx_generator import actions
+from tests.tx_generator import config as conf
+from tests.tx_generator.models.wallet_api import WalletAPI
+from tests.tx_generator.models.accountant import Accountant
+from tests.utils import print_hits_entry_count
+
+
+def test_grpc_stress(init_session, setup_network):
+    # use the same node to send all api calls (-1)
+    wallet_api = WalletAPI(init_session, setup_network.clients.pods, -1)
+    accountant = Accountant({conf.acc_pub: Accountant.set_tap_acc()})
+    api_funcs_lst = [wallet_api.get_balance_value, wallet_api.get_nonce_value, wallet_api.get_tx_by_id,
+                     actions.transfer]
+
+    # create #new_accounts accounts, this will also be the number of
+    # concurrent api calls
+    new_accounts = 50
+    amount = 50
+    print(f"\n\n----- create {new_accounts} new accounts with {amount} coins for each -----")
+    actions.send_coins_to_new_accounts(wallet_api, new_accounts, amount, accountant)
+
+    layer_duration = int(testconfig["client"]["args"]["layer-duration-sec"])
+    tts = layer_duration * conf.num_layers_until_process
+    sleep_print_backwards(tts)
+
+    processes = []
+    queue = mp.Queue()
+    for x in range(new_accounts):
+        # get acc pub
+        acc_pub = list(accountant.accounts.keys())[x]
+        # select one of the api calls to run randomly
+        func = random.choice(api_funcs_lst)
+        args = (acc_pub, )
+        if func == actions.transfer:
+            dst = random.choice(list(accountant.accounts.keys()))
+            amount = 1
```

Comment:
why only 1? can we have a random value for this?

Reply:
sure

Reply:
1153c9903d5bb035b62181f7dd46a0eb247c5639